ctags/main/entry.c

```diff
@@ -387,7 +387,7 @@ extern void makeTagEntry (const tagEntryInfo *const tag)
 {
 	Assert (tag->name != NULL);
 	if (tag->name [0] == '\0')
-		error (WARNING, "ignoring null tag in %s", vStringValue (File.name));
+		error (WARNING, "ignoring null tag in %s", getInputFileName ());
```

Comment:
hum no, it actually doesn't: it matches the call, but the implementation in UCTags is `return vStringValue (File.input.name);`, while ours (see below) is `vStringValue (File.source.name)`.
I have no idea if it changes anything in practice, but it's not the same.